app/views/shared/_navbar.html.erb

```diff
@@ -31,6 +31,9 @@
           <li class="nav-item">
             <%= link_to "Sign Up", new_user_registration_path, class: "nav-link" %>
           </li>
+          <li class="nav-item">
+          <%= link_to "Add Item", new_equipment_item_path, class: "nav-link" %>
```

Comment:
can you move this to the already signed in nav bar so line 24 ish

Reply:
Fixed this, check path please.